src/components/CertificateViewer.tsx

```diff
@@ -34,7 +34,9 @@ export const CertificateViewer: FunctionComponent<CertificateViewerProps> = ({ d
       trace(e);
     }
   }
-  const tokenRegistryAddress = isTransferableAsset ? utils.getIssuerAddress(document)[0] : "";
+
+  const issuerAddress = isTransferableAsset ? utils.getIssuerAddress(document) : "";
+  const tokenRegistryAddress = issuerAddress instanceof Array ? issuerAddress[0] : issuerAddress;
```

Comment:
instead of doing this here, can we abstract into a function in utils/shared?

Reply:
okay, i have extracted into getTokenRegistryAddress